tests/test_pangonet_distance.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from pathlib import Path
+import numpy as np
+import sys
+sys.path.append("../notebooks/")
+import nb_utils
+
+
+class TestPangonetDistance():
+    def start_pangonet(self):
+        rebar_dir = Path("../notebooks/dataset/rebar")
+        alias_key_file = rebar_dir / "alias_key.json"
+        lineage_notes_file = rebar_dir / "lineage_notes.txt"
+        return nb_utils.initialise_pangonet(
+            alias_key_file=alias_key_file,
+            lineage_notes_file=lineage_notes_file,
+        )
+
+
+    @pytest.mark.parametrize(
+        "labels, expected",
+        [
+            # Special case
+            (("BM.1", "BM.1"), 0),
+            # Parent-child pair
+            (("BM.1", "BM.1.1"), 1),
+            # Ancestor-descendant pair
+            (("BM.1", "BM.1.1.1"), 2),
+            # Sibling pair
+            (("BM.1", "BJ.1"), 6),
+            # Uncompressed/compressed pair
+            (("B.1.1.529.2.75.3", "BM.1"), 1),
+        ],
+    )
+    def test_pangonet_distance(self, labels, expected):
+        pangonet = self.start_pangonet()
+        actual = nb_utils.get_pangonet_distance(
+            pangonet=pangonet,
+            label_1=labels[0],
+            label_2=labels[1],
+        )
+        np.testing.assert_equal(actual, expected)
```

Comment:
There's no need to use ``assert_equal`` here, the results are scalars right? ``assert actual == expected`` is better

Reply:
Done